dev-python/dissononce/dissononce-0.34.3.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A python implementation for Noise Protocol Framework"
+HOMEPAGE="https://github.com/tgalal/dissononce"
+SRC_URI="https://github.com/tgalal/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
This looks wrong, I see MIT in their github page?

Reply:
Good catch, I must overseen this. Changed.